gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixRetriggeringJobCallable.java

```diff
@@ -249,8 +260,9 @@ private void runJobExecutionLauncher() throws JobException {
     String newPlanningId;
     Closer closer = Closer.create();
     try {
-      HelixManager planningJobManager = this.taskDriverHelixManager.isPresent()?
+      HelixManager planningJobHelixManager = this.taskDriverHelixManager.isPresent()?
           this.taskDriverHelixManager.get() : this.jobHelixManager;
```

Comment:
`getOrElse`?

...actually thought I earlier saw a similar replacement above...

Reply:
Yes, i removed it in this PR. 1st change of the PR